canonical-web-and-design/ubuntu.com
# Master rebase

## Done

Rebase focal branch from master

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure that all the 20.04 changes still look good


## Issue / Card

Fixes #7283
